version.h

```diff
@@ -146,6 +161,62 @@
 #define kCFCoreFoundationVersionNumber_iOS_11_2 1450.14
 #endif
 
+#ifndef kCFCoreFoundationVersionNumber_iOS_11_3
+#define kCFCoreFoundationVersionNumber_iOS_11_3 1452.23
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_11_4
+#define kCFCoreFoundationVersionNumber_iOS_11_4 kCFCoreFoundationVersionNumber_iOS_11_3
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_12_0
+#define kCFCoreFoundationVersionNumber_iOS_12_0 1556.00
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_12_1
+#define kCFCoreFoundationVersionNumber_iOS_12_1 1560.10
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_12_2
+#define kCFCoreFoundationVersionNumber_iOS_12_2 1570.15
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_12_3
+#define kCFCoreFoundationVersionNumber_iOS_12_3 1575.13
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_12_4
+#define kCFCoreFoundationVersionNumber_iOS_12_4 1575.17
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_13_0
+#define kCFCoreFoundationVersionNumber_iOS_13_0 1665.15
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_13_1
+#define kCFCoreFoundationVersionNumber_iOS_13_1 1671.101
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_13_2
+#define kCFCoreFoundationVersionNumber_iOS_13_2 1673.126
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_13_3
+#define kCFCoreFoundationVersionNumber_iOS_13_3 1674.102
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_13_4
+#define kCFCoreFoundationVersionNumber_iOS_13_4 1675.129
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_13_5
+#define kCFCoreFoundationVersionNumber_iOS_13_5 1676.104
+#endif
+
+#ifndef kCFCoreFoundationVersionNumber_iOS_14_0
+#define kCFCoreFoundationVersionNumber_iOS_14_0 1740.00
```

Comment:
Based on convention used in the rest of the file, this should match the exact CF version number rather than the lower bound. But I think it’ll be fine if we keep this as-is and change it when the GM comes out.

Reply:
Agreed.